css/style.css

```diff
@@ -464,6 +500,17 @@ input::placeholder {
     padding: 60px 120px 24px 120px;
   }
 
+  #current-time {
+    margin-left: 120px;
+  }
+
+  .footer {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+  }
```

Comment:
- [ ] You have done a great job adding the footer to your app, however the footer now obstructs the smooth usage of your app because it covers the submit button on desktop devices as you can see below 👇 Please try to fix this design problem in order to get approved 👏 
- NOTE: This problem affects the Books list and Contact info sections as well, so kindly fix it 👍



![image](https://user-images.githubusercontent.com/46839548/204907267-fc923b06-6580-430b-964c-c748d682d496.png)